Gemfile.lock

```diff
@@ -75,6 +80,10 @@ GEM
     erubis (2.7.0)
     fakeredis (0.5.0)
       redis (~> 3.0)
+    faraday (0.9.2)
+      multipart-post (>= 1.2, < 3)
+    faraday_middleware (0.10.0)
```

Comment:
it is being used here https://github.com/department-of-veterans-affairs/vets-api/pull/41/files/348857851cd8b2b298935fb1f49c5ee665a30603#diff-23334900c30744ee4fd5c694c1cece87R28